ray-operator/controllers/ray/utils/dashboard_httpclient.go

```diff
@@ -222,6 +223,7 @@ type RayJobLogsResponse struct {
 }
 
 // Note that RayJobInfo and error can't be nil at the same time.
```

Comment:
TODO: KubeRay uses many different methods to create `errors`. We should try to unify them.
* "k8s.io/apimachinery/pkg/api/errors"
* "github.com/pkg/errors"
* `fmt.Errorf`